packages/yuudachi/src/interactions/moderation/lockdown.ts

```diff
@@ -34,6 +34,8 @@ export const LockdownCommand = {
 					name: "reason",
 					description: "The reason of this lockdown",
 					type: ApplicationCommandOptionType.String,
+					min_length: 3,
+					max_length: 500,
```

Comment:
Still missing the imports here?

Reply:
Tbh, forgot this have a reason, will push a fix

Reply:
Fixed in 05796ee